### PR TITLE
Some bootstrap changes

### PR DESCRIPTION
**Pull request by balen**
I fixed some of the bootstrap
- added a floating button for the mail my ballot
- header and footers are afixed to top and bottom
- some html cleanup
- and a docker compose for local dev

### CODE REVIEW

File: nominate/templates/nominate/nominate.html [hunk 1]
```diff
@@ -4,43 +4,56 @@
...
-                    <div class="col-12 col-lg-3">
+                    <div class="d-flex-row">{% include "nominate/bits/nomination_instructions.html" %}</div>
+                    <!-- <div class="col-12 col-lg-3">
                         <form action="{% url 'election:email-my-nominations' election_id=election.slug %}"
                               method="post">
                             {% csrf_token %}
                             <button type="submit" class="btn btn-primary">{% translate "Email my nominations to me" %}</button>
                         </form>
-                    </div>
+                    </div> -->
```
Comment (offbyone) [inline nominate/templates/nominate/nominate.html:28]: No sense in leaving this commented code in, unless you have a specific reason.

File: glasgow2024/templates/bits/convention_header.html [hunk 2]
```diff
@@ -1,50 +1,42 @@
...
-                    {% endif %}
-                </ul>
-            </div>
-        </div>
+      <div class="collapse navbar-collapse bg-light" id="navbarContent">
+        <div class="navbar-text mx-auto center-title">#Glasgow2024</div>
+        <ul class="navbar-nav mr-auto">
+          {% if request.user.is_staff %}
+          <li class="nav-item">
+            <a class="nav-link" href="{% url " admin:index" %}">
```
Comment (offbyone) [inline glasgow2024/templates/bits/convention_header.html:18]:
```suggestion
            <a class="nav-link" href="{% url "admin:index" %}">
```

It looks like _I_ missed this, somehow. I'll have to figure out why. In the meantime, mind including it?

**Review by offbyone — CHANGES_REQUESTED** [review]

**Comment by offbyone** [comment]
Barring the dev process side, I like the layout changes; happy to merge those.

File: docker-compose-dev.yml
[the remark this answers is not in the document]
Comment (balen) [reply docker-compose-dev.yml:1]: the three files have been removed as part of this commit
FYI - I left the .envrc addition to gitignore as I and others often use that for env as well and it should not be checked into a project

File: nominate/templates/nominate/nominate.html (hunk 1 above)
Comment (balen) [reply nominate/templates/nominate/nominate.html:28]: Yep, makes sense to me

**Pull request merged**